timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -81,33 +97,173 @@ limitations under the License.
+              </td>
+            </tr>
+            <tr v-for="(item, key) in getAnalyzerOutputMetaData" :key="key">
+              <td style="border: none">
+                <strong>{{ key }}:</strong>
+              </td>
+              <td style="border: none" v-if="key === 'Searches'">
+                <v-chip v-for="(view, index) in item" :key="index" @click="setView(view)" outlined class="mr-1" small>
+                  {{ view.name }}
+                </v-chip>
+              </td>
+              <td style="border: none" v-if="key === 'Stories'">
+                <router-link
+                  v-for="story in item"
+                  :key="story.id"
+                  :to="{ name: 'Story', params: { storyId: story.id } }"
+                >
+                  <v-chip outlined class="mr-1" small link>{{ story.title }}</v-chip>
+                </router-link>
+              </td>
+              <td style="border: none" v-if="key === 'Graphs'">
+                <router-link
+                  v-for="(graph, index) in item"
+                  :key="index"
+                  :to="{ name: 'Graph', params: { graph: graph.id === undefined ? graph.name : graph.id } }"
+                >
+                  <v-chip
+                    outlined
+                    class="mr-1"
+                    small
+                    link
+                    @click="graph.id === undefined ? setGraphPlugin(graph.name) : setSavedGraph(graph.id)"
+                  >
+                    {{ graph.id === undefined ? graph.display_name : graph.name }}
+                  </v-chip>
+                </router-link>
+              </td>
+              <!-- TODO(jkppr): Add Aggregations as soon as they are supported in the new UI -->
+              <td style="border: none" v-if="key === 'Aggregations'">
+                <span>Linking aggregations is not supported in the new UI yet.</span>
+              </td>
+              <td style="border: none" v-if="key === 'Tags'">
+                <v-chip
+                  v-for="(tag, index) in item"
+                  :key="index"
+                  color="lightgrey"
+                  class="mr-1 mb-1"
+                  small
+                  @click="searchForTag(tag)"
+                >
+                  {{ tag }}
+                </v-chip>
+              </td>
+            </tr>
+          </tbody>
+        </v-simple-table>
+        <v-simple-table v-else dense class="ml-2">
+          <tbody
+            :class="
+              $vuetify.theme.dark
+                ? expanded
+                  ? 'dark-hover dark-bg'
+                  : 'dark-hover'
+                : expanded
+                ? 'light-hover light-bg'
+                : 'light-hover'
+            "
+          >
+            <tr class="pr-3">
+              <td width="80" style="border: none">
+                <strong v-if="timeline.analysis_status === 'ERROR'">Error:</strong>
+                <strong v-else>Result:</strong>
```

Comment:
Maybe call this `Verdict` instead?

Reply:
I think I will go with `Summary` first and collect user feedback on the chosen wording.